src/store/Settings.ts

```diff
@@ -29,23 +30,32 @@ const types = flow(
     'SET_WALLET_LOADED',
     'SET_NETWORK_FEES',
     'RESET_RUNTIME_VERSION_SUBSCRIPTION',
+    'SET_NFT_STORAGE',
+    'SET_API_KEYS',
   ]),
   map((x) => [x, x]),
   fromPairs
 )([]);
 
+export type ApiKeysObject = {
```

Comment:
Maybe we should add this type to `WALLET_TYPES ` to reuse it polkaswap?

Reply:
Moved to WALLET_TYPES